### PR TITLE
docs: small fixes in  upgrade responses.rst

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- change page title
- fix typo

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

**Review by datamweb — APPROVED** [review]

File: user_guide_src/source/installation/upgrade_4xx.rst [hunk 1]
```diff
@@ -254,8 +254,8 @@ Upgrading Libraries
     upgrade_images
     upgrade_localization
     upgrade_migrations
-    upgrade_pagination
     upgrade_responses
+    upgrade_pagination
```
Comment (paulbalandan) [inline user_guide_src/source/installation/upgrade_4xx.rst:258]: Why this change? I assume this was alphabetical.

Comment (kenjis) [reply user_guide_src/source/installation/upgrade_4xx.rst:258]: Because I changed the page title.
Screenshot 2024-04-23 6 26 33

Comment (michalsn) [reply user_guide_src/source/installation/upgrade_4xx.rst:258]: The output class makes more sense from the perspective of a CI3 user.

**Review by michalsn — APPROVED** [review]
